src/game.cpp

```diff
@@ -8414,16 +8438,8 @@ void game::reload( item_location &loc, bool prompt, bool empty )
     }
 
     // for holsters and ammo pouches try to reload any contained item
```

Comment:
Will this still be a useful behavior with the new inventory interface?

Reply:
i decided to keep it in because of discussion about holster UX.